sharkfuser/include/fusili/attributes.h

```diff
@@ -0,0 +1,188 @@
+// Copyright 2025 Advanced Micro Devices, Inc.
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
```

Comment:
Can we stick to IREE code style? IREE C++ codestyle is derived from LLVM. We use header guards: https://llvm.org/docs/CodingStandards.html#header-guard

Reply:
@Groverkss Header guard changes are in. I'm using LLVM style clang-format. Btw [noticed](https://sourcegraph.com/github.com/iree-org/iree/-/blob/.clang-format?L9) IREE uses Google style clang-format (there's a comment about runtime using Google vs compiler using LLVM style).